simple-local-avatars.php

```diff
@@ -134,11 +152,38 @@ public function get_avatar( $avatar = '', $id_or_email = '', $size = 96, $defaul
 
 		if ( 'http' != substr( $local_avatars[$size], 0, 4 ) )
 			$local_avatars[$size] = home_url( $local_avatars[$size] );
-		
-		$author_class = is_author( $user_id ) ? ' current-author' : '' ;
-		$avatar = "<img alt='" . esc_attr( $alt ) . "' src='" . esc_url( $local_avatars[$size] ) . "' class='avatar avatar-{$size}{$author_class} photo' height='{$size}' width='{$size}' />";
-		
-		return apply_filters( 'simple_local_avatar', $avatar );
+
+		return esc_url( $local_avatars[$size] );
+	}
+
+	/**
+	 * Get default avatar url 
+	 *
+	 * @since 2.2.0
+	 *
+	 * @param int $size Requested avatar size. 
+	 */
+	public function get_default_avatar_url( $size ) {
+		if ( empty( $default ) ) {
+			$avatar_default = get_option( 'avatar_default' );
+			if ( empty( $avatar_default ) )

```

Comment:
I know this was just copied from the current codebase but we should make sure any if statements here have brackets `{ }` at the end of the lines.